zerver/models.py

```diff
@@ -257,6 +258,14 @@ def get_signup_notifications_stream(self) -> Optional['Stream']:
             return self.signup_notifications_stream
         return None
 
+    # The quota is stored in gibibytes "GiB" as the difference is immaterial and
+    # it's a bit more generous than gigabytes "GB". We would still use gigabytes
+    # in user facing text, because otherwise we have to explain what a gibibyte is.
```

Comment:
Hrm, I tried a somewhat tricky bit of workflow in the last round: I pushed directly to your branch a version that had some small changes of mine (the kind of changes I would make before merging if I were otherwise ready to merge.) Clearly that was too tricky for me to do without explaining more!

I'm still interested in taking a shot at a fancier workflow sometime, but for now: would you squash this diff into the commit that adds this? (This is just an updated version of the change I pushed last time.)
```
--- a/zerver/models.py
+++ b/zerver/models.py
@@ -174,6 +174,8 @@ class Realm(models.Model):
     DEFAULT_MAX_INVITES = 100
     max_invites = models.IntegerField(default=DEFAULT_MAX_INVITES)  # type: int
     message_visibility_limit = models.IntegerField(null=True)  # type: int
+
+    # See upload_quota_bytes; don't interpret upload_quota_gb directly.
     upload_quota_gb = models.IntegerField(null=True)  # type: int
 
     # Define the types of the various automatically managed properties
@@ -258,12 +260,11 @@ class Realm(models.Model):
             return self.signup_notifications_stream
         return None
 
-    # The quota is stored in gibibytes "GiB" as the difference is immaterial and
-    # it's a bit more generous than gigabytes "GB". We would still use gigabytes
-    # in user facing text, because otherwise we have to explain what a gibibyte is.
     def upload_quota_bytes(self) -> Optional[int]:
         if self.upload_quota_gb is None:
             return None
+        # We describe the quota to users in "GB" or "gigabytes", but actually apply
+        # it as gibibytes (GiB) to be a bit more generous in case of confusion.
         return self.upload_quota_gb << 30
 
     @property
```